docs/wiki/User Guide/Configurations.md

```diff
@@ -208,9 +208,29 @@ We are still trying to improve cruise control. And following are some configurat
 | skip.sample.store.topic.rack.awareness.check    | Boolean |  N         |   false           | The config to skip rack awareness sanity check for sample store topics  |
 
 ### BrokerCapacityConfigurationFileResolver configurations
-| Name                 | Type   | Required? | Default Value | Description                                                                        |
-|----------------------|--------|-----------|---------------|------------------------------------------------------------------------------------|
-| capacity.config.file | String | Y         |               | The path to the configuration JSON file that provides the capacity of the brokers. |
+| Name                 | Type   | Required? | Default Value             | Description                                                                        |
+|----------------------|--------|-----------|---------------------------|------------------------------------------------------------------------------------|
+| capacity.config.file | String | Y         | config/capacityJBOD.json  | The path to the configuration JSON file that provides the capacity of the brokers. |
+
+#### Populating the Capacity Config File
+`Option-1`: The following steps allow users to set the initial broker capacities or update capacities upon addition of 
+new brokers out-of-the-box:
+1. Check `cruisecontrol.properties` file to verify that the `capacity.config.file=` config points to the file you are modifying.
+By default, this config is set to use `config/capacityJBOD.json`. If you have 
+    * a non-JBOD deployment, where each broker has the same number of cores, you may use `config/capacity.json`, or
+    * a non-JBOD deployment, where brokers may have varying number of cores, you may use `config/capacityCores.json`
+ and modify this file.
+2. Ensure that the capacity file picked in `step-1` contains either a default capacity for missing broker ids, or 
+explicitly specifies capacities for brokers with specific ids provided by user.
+3. Finally, bounce your Cruise Control instance.
+
+`Option-2`: `BrokerCapacityConfigResolver` is a pluggable component. Hence, you can write your own pluggable capacity 
```

Comment:
I feel some context is missing here. option-1 is only relevant with the default implementation mentioned in option-2, should we first give some background first, then list two options
1. stick with default resolver and config the capacity files.
2. implement your own resolver.